is_iot_sink/allowed_collectors.py

```diff
@@ -8,14 +8,24 @@ class AllowedCollectors:
     def __init__(self):
         self.__collectors = {}
         self.__expire_time = int(utils.get_setting("collectors/expireTime"))
+        self.__registration_enabled = True if utils.get_setting("collectors/registrationEnabled").lower() == "true" else False
         self.__check_thread = threading.Thread(target=self.__check_all, daemon=True)
         self.__check_thread.start()
 
+    def is_registration_enabled(self):
+        return self.__registration_enabled
+
+    def is_registration_disabled(self):
```

Comment:
A bit overkill to have is_enabled() and is_disabled() as 2 separate functions that emulate pretty much the same functionality.
Don't you think that just having the first option and using it with not when necesarry is the cleaner option?

Reply:
I agree with @bkpnebunu here. It just does not make sense to have 2 separate functions.

Reply:
Thanks for your time reviewing this, but I do not agree with both of you.

Imagine having 100 `if` branches where I want to check if the registration is enabled. I would type simply `if is_registration_enabled():`. But if I have another 100 `if` branches where I want to check if the registration is disabled, I would have to type the `not` or `!` 100 times like `if not is_registration_enabled():`.

So I type `not` only in one place and am happy that I have clear functions and not complicating conditions. It is cleaner to read the function name and know directly what it does.

Take a look at how ruby implements the `present?` function https://apidock.com/rails/Object/present%3F.

Reply:
He's a ruby fanboy now. I'll allow it.
Il dor destele daca scrie prea mult. :rofl:

Reply:
I still feel like it's code duplication as both functions do the same thing.
I understand your reason, but having 2 different, but similarly named functions, checking the same functionality, and going forward with your example of 100 `if` branches where you have to check, makes mistakes where you mess their names up while writing code way more easy to make than having a `not` keyword written.
But this is just a personal opinion for both of us i believe and that it can be merged as it is from my side.

Reply:
:face_with_head_bandage: :open_hands: